app/items/[uuid]/page.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Metadata } from "next";
+import PageLayout from "app/components/pageLayout/pageLayout";
+import Item from "@/components/items/item";
+import { getItemData } from "@/utils/utils";
+import { ItemModel } from "@/models/item";
+
+type ItemProps = {
+  params: {
+    uuid: string;
+    item: ItemModel;
+  };
+};
+
+const getItem = async (uuid) => {
```

Comment:
Regarding your question in the description, looks like it's a common problem and request: https://github.com/vercel/next.js/discussions/50080

It doesn't seem like a big problem though but it's unfortunate.